Travel Scheduler/Test/Schedule.m

```diff
@@ -7,70 +7,269 @@
+        return 1;
+    } else if ([dayString isEqualToString:@"Tuesday"]) {
+        return 2;
+    } else if ([dayString isEqualToString:@"Wednesday"]) {
+        return 3;
+    } else if ([dayString isEqualToString:@"Thursday"]) {
+        return 4;
+    } else if ([dayString isEqualToString:@"Friday"]) {
+        return 5;
+    } else if ([dayString isEqualToString:@"Saturday"]) {
+        return 6;
+    } else if ([dayString isEqualToString:@"Sunday"]) {
+        return 0;
+    }
+    return -1;
+}
+
+static void getDistanceToHome(Place *place, Place *home) {
+    dispatch_semaphore_t sema = dispatch_semaphore_create(0);
+    [[APIManager shared] getDistanceFromOrigin:place.name toDestination:home.name withCompletion:^(NSDictionary *distanceDurationDictionary, NSError *error) {
+        if (distanceDurationDictionary) {
+            NSNumber *time = [distanceDurationDictionary valueForKey:@"value"][0][0];
+            place.travelTimeFromPlace = time;
+        } else {
+            NSLog(@"Error getting closest place: %@", error.localizedDescription);
+        }
+        dispatch_semaphore_signal(sema);
+    }];
+    dispatch_semaphore_wait(sema, DISPATCH_TIME_FOREVER);
+}
 
 @implementation Schedule
 
 #pragma mark - initialization methods
+
 - (instancetype)initWithArrayOfPlaces:(NSArray *)completeArrayOfPlaces withStartDate:(NSDate *)startDate withEndDate:(NSDate *)endDate {
     self = [super init];
     [self createAllProperties];
-    [self.arrayOfAllPlaces arrayByAddingObjectsFromArray:completeArrayOfPlaces];
-    [self createAvaliabilityDictionary];
     self.startDate = startDate;
     self.endDate = endDate;
     self.numberOfDays = (int)[Date daysBetweenDate:startDate andDate:endDate];
+    
+    
+
+    [self testing];
+    self.startDate = removeTime([NSDate date]);
+    self.endDate = removeTime(getNextDate(self.startDate, 2));
+    //TODO: do it with real arrays.
+    //[self.arrayOfAllPlaces arrayByAddingObjectsFromArray:completeArrayOfPlaces];
+    //self.home = [self.arrayOfAllPlaces objectAtIndex:0];
+    //[self createAvaliabilityDictionary];
+    
+    //self.numberOfDays = (int)[Schedule daysBetweenDate:startDate andDate:endDate];
+    
     return self;
 }
 
-#pragma mark - helper methods for initialization
-- (void)createAllProperties {
-    self.availabilityDictionary = [[NSMutableDictionary alloc] init];
-    self.availabilityDictionary[@"breakfast"] = [[NSMutableDictionary alloc] init];
-    self.availabilityDictionary[@"morning"] = [[NSMutableDictionary alloc] init];
-    self.availabilityDictionary[@"lunch"] = [[NSMutableDictionary alloc] init];
-    self.availabilityDictionary[@"afternoon"] = [[NSMutableDictionary alloc] init];
-    self.availabilityDictionary[@"dinner"] = [[NSMutableDictionary alloc] init];
-    self.availabilityDictionary[@"evening"] = [[NSMutableDictionary alloc] init];
-    self.arrayOfAllPlaces = [[NSMutableArray alloc] init];
+- (void)testing {
```

Comment:
I think it would be good to put the testing methods in its own separate file

Reply:
Yeah, I'll do that.